src/features/calendar/l10n/messageIds.ts

```diff
@@ -71,6 +71,7 @@ export default makeMessages('feat.calendar', {
   ranges: {
     day: m('Day'),
     month: m('Month'),
+    shortWeek: m('w'),
```

Comment:
The strings here (`ranges`) are the terms used in the daterange picker in the top right of the calendar, where the user can switch between viewing the calendar in "month", "week" or "day" view.

This new string doesn't really have anything to do with that switcher, so I think it should live somewhere else in the messages structure, maybe at the top level?